components/html-embed/index.js

```diff
@@ -0,0 +1,37 @@
+export default class HtmlEmbed extends wp.element.Component {
+
+	static get defaultProps() {
+		return {
+			html: '',
+		};
+	}
+
+	componentDidMount() {
+		const body = this.node;
+		const { html } = this.props;
+
+		body.innerHTML = html;
+
+		const scripts = body.getElementsByTagName( 'script' );
+		const newscripts = [];
+
+		for ( let i = 0; i < scripts.length; i++ ) {
```

Comment:
Minor: could this be a map function `const newscripts = Array.from( scripts ).map( ... )`?